src/views/AppRoot/__tests__/container.spec.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import configureStore from 'redux-mock-store'
+import { Provider } from 'react-redux'
+import { MemoryRouter } from 'react-router-dom'
+
+import AppRoot from '../container'
+import LoginContainer from '../../pages/Login'
+
+describe('AppRoot match snapshot', () => {
+  let mockStore
+
+  beforeAll(() => {
+    mockStore = configureStore([])
+  })
+
+  it('with default props', () => {
+    const container = mount(
+      <Provider store={mockStore({ session: { isLogged: true } })}>
+        <MemoryRouter initialEntries={['/']}>
```

Comment:
create some helper function to setup all wrappers like `Provider, MemoryRouter` etc